### ISSUE TITLE
Add qemu-tcmu support for gluster-block tool

### ISSUE THREAD

**Issue by lxbsz** [gluster/gluster-block#59]
Currently, the qemu-tcmu is under 3rd Version by the community: https://github.com/open-iscsi/tcmu-runner/pull/263, and now is blocked by the License issue.

**Comment by amarts** [comment]
@pkalever @lxbsz @pranithk can we pick the code locally, and try running some basic tests while we fix the license issues?

Specifically, the performance penalty we would get because of qcow2 format, and specially after taking some snapshots.

**Comment by lxbsz** [comment]
@amarts yes, sure, I will do this.
Before I have test the snapshot, it still couldn't work though it has already has the feature support.

**Comment by pkalever** [comment]
Won't Fix, closing now.
